### PR TITLE
add time and account name

### PR DESCRIPTION
**Pull request by cigra**

### CODE REVIEW

File: armotypes/usernotificationreporttypes.go [hunk 1]
```diff
@@ -6,6 +6,9 @@ import (
 )
 
 type WeeklyReport struct {
+	From                                time.Time                `json:"from" bson:"from"`
```
Comment (RinaO1234) [inline armotypes/usernotificationreporttypes.go:9]: can we add unit tests

Comment (cigra) [reply armotypes/usernotificationreporttypes.go:9]: done

File: armotypes/usernotificationreporttypes_test.go [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
 
+	_ "embed"
 	"github.com/stretchr/testify/assert"
 )
 
+//go:embed testdata/weeklyReport.json
+var weeklyReport string
+
+func TestWeeklyReport(t *testing.T) {
+	from, _ := time.Parse(time.RFC3339,  "2023-01-07T00:00:00+00:00")
```
Comment (RinaO1234) [inline armotypes/usernotificationreporttypes_test.go:16]: let's add assertNoError

File: armotypes/usernotificationreporttypes_test.go [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
+		LinkToConfigurationScanningFiltered: "http://somelink1.com",
+		RepositoriesScannedThisWeek:         3,
+		RepositoriesScannedPrevWeek:         4,
+		LinkToRepositoriesScanningFiltered:  "http://somelink2.com",
+		RegistriesScannedThisWeek:           5,
+		RegistriesScannedPrevWeek:           6,
+		LinkToRegistriesScanningFiltered:    "http://somelink3.com",
+		Top5FailedControls:                  []TopCtrlItem{{Name: "control1", TotalFailedResources: 1}},
+	}
+	b, _ := json.Marshal(report)
```
Comment (RinaO1234) [inline armotypes/usernotificationreporttypes_test.go:33]: same here

Comment (cigra) [reply armotypes/usernotificationreporttypes_test.go]: done

**Review by RinaO1234 — APPROVED** [review]

**Review by RinaO1234 — APPROVED** [review]

**Pull request merged**